### PR TITLE
feat(apps): add MuseScore

### PR DESCRIPTION
**Pull request by cubeDHS2017**
I added a rule for MuseScore that would overflow it's borders.

- [ ] I have formatted `applications.yaml` with `komorebic fmt-asc`.

### CODE REVIEW

File: applications.yaml [hunk 1]
```diff
@@ -1459,3 +1459,17 @@
   - kind: Exe
     id: ueli.exe
     matching_strategy: Equals
+- name: Morgen
+  identifier:
+    kind: Exe
+    id: Morgen.exe
+    matching_strategy: Equals
+  options:
+  - border_overflow
```
Comment (sitiom) [inline applications.yaml:1468]:
```suggestion
```
Please clean out other unnecessary changes before making your PR.

File: applications.yaml [hunk 2]
```diff
@@ -1459,3 +1459,17 @@
...
+- name: MuseScore
+  identifier:
+    kind: Exe
+    id: MuseScore.exe
+    matching_strategy: Equal
```
Comment (sitiom) [inline applications.yaml:1473]:
```suggestion
    matching_strategy: Equals
```

**Review by sitiom — CHANGES_REQUESTED** [review]

**Comment by sitiom** [comment]
Your commit history seems to be dirty. I recommend cleaning up your master branch and creating your PRs on a separate branch to avoid unexpected changes being introduced to your PR. See: [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow)

**Comment by cubeDHS2017** [comment]
> Your commit history seems to be dirty. I recommend cleaning up your master branch and creating your PRs on a separate branch to avoid unexpected changes being introduced to your PR. See: [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow)

How would I better setup my workflow for this project and making PRs?

**Comment by sitiom** [comment]
1. Delete and recreate your fork to clean up your master branch (or clean it up with a bunch of commands if you know how 🙂)
2. From the master branch, create a new branch based on the new app you want to add (Example: `musescore`)
image
3. Make your changes from there, run `komorebic fmt-asc applications.yaml`, and open a PR based on that branch.
4. Once your PR is merged, delete the branch, go back to the master branch, and sync the upstream changes from your fork.

I recommend learning the basics of Git to get started: https://docs.github.com/en/get-started/quickstart/git-and-github-learning-resources
